boards/nuvoton/numaker_m55m1/numaker_m55m1.dts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2025 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nuvoton/m55m1h2l.dtsi>
+#include "numaker_m55m1-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Nuvoton NuMaker M55M1 board";
+	compatible = "nuvoton,numaker-m55m1";
+
+	aliases {
+		led0 = &green_led;
+		led1 = &yellow_led;
+	};
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,dtcm = &dtcm;
+		zephyr,itcm = &itcm;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led: led_0 {
+			gpios = <&gpiod 5 GPIO_ACTIVE_LOW>;
+			label = "User LD0";
+		};
+
+		yellow_led: led_1 {
+			gpios = <&gpiod 6 GPIO_ACTIVE_LOW>;
+			label = "User LD1";
+		};
+	};
+};
+
+&gpiod {
+	status = "okay";
+};
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x0 0x8000>;
+		};
+		slot0_partition: partition@8000 {
```

Comment:
newline separations needed between these

Reply:
Thank your advice and done